share/man/man7/ports.7

```diff
@@ -598,22 +613,22 @@ The big Kahuna.
 .El
 .Sh EXAMPLES
 .Bl -tag -width 0n
-.It Sy Example 1\&: No Building and Installing a Port
+.It Sy Example 1\&: Building and Installing a Port
 .Pp
 The following command builds and installs Emacs.
 .Bd -literal -offset 2n
-.Li # Ic cd /usr/ports/editors/emacs
-.Li # Ic make install
+.Ic # cd /usr/ports/editors/emacs
+.Ic # make install
 .Ed
-.It Sy Example 2\&: No Installing Dependencies with Xr pkg 8
+.It Sy Example 2\&: Installing Dependencies with Xr pkg 8
```

Comment:
On the contrary, references to `pkg(7)` should be changed to `pkg(8)` unless they're specifically about the bootstrap tool present in base rather than the actual package management tool installed from ports.

Reply:
Ok. This was unclear because `pkg-static` leads to pkg(8). Should we switch pkg-static to pkg(7)?